post_compile

```diff
@@ -1,29 +1,45 @@
 #!/usr/bin/env bash
 
-set -e
+set -eo pipefail
+
+indent() {
+  sed "s/^/       /"
+}
+
+puts-step() {
+  echo "-----> $*"
+}
+
+puts-warn() {
+  echo " !     $*"
+}
```

Comment:
Why we can't just source those from `bin/utils.sh` as well?

Reply:
Because this will be executed inside the repo when the build packs is no longer available.

Reply:
@codingjoe right, I've got it. Thank you :)